templates/rhpam74-prod-immutable-kieserver-amq.yaml

```diff
@@ -324,6 +325,31 @@ parameters:
   name: AMQ_GLOBAL_MAX_SIZE
   example: 10 gb
   required: false
+- description: "Name of a secret containing SSL related files."
+  displayName: Secret Name
+  name: AMQ_SECRET
```

Comment:
provide a default value like KIE_SERVER_HTTPS_SECRET
This will fix the following issue reported by ci:
```
Errors found: {
  "DeploymentConfig-myapp-amq": [
    "spec.template.spec.volumes[0].secret.secretName: Required value",
    "spec.template.spec.containers[0].volumeMounts[0].name: Not found: \"broker-secret-volume\""
  ]
}

```